README.md

```diff
@@ -492,6 +492,15 @@ EmberCLI runners to clobber each others' work][#94].
 [Unicorn]: https://rubygems.org/gems/unicorn
 [#94]: https://github.com/thoughtbot/ember-cli-rails/issues/94#issuecomment-77627453
 
+## `EMBER_ENV`
+
+If set on the environment, the value of `EMBER_ENV` will be passed to the
+`ember` process as the value of the `--environment` flag.
```

Comment:
I'm a bit confused on this. Doesn't (this code)[https://github.com/thoughtbot/ember-cli-rails/blob/v0.7.0/lib/ember_cli/command.rb#L50] always force the environment passed to `--environment` to be either `development` or `production`?